Microsoft.Recognizers.Text.NumberWithUnit/English/Extractors/CurrencyExtractorConfiguration.cs

```diff
@@ -0,0 +1,513 @@
+                "cook islands dollars|cook islands dollar|cook islands $|cook island dollars|cook island dollar|cook island $"
+            },
+            {"Fijian dollar", "fijian dollars|fijian dollar|fijian $|fjd|fiji dollars|fiji dollar|fiji $"},
+            {"Guyanese dollar", "guyanese dollars|guyanese dollar|gyd|gy$"},
+            {
+                "Hong Kong dollar",
+                "hong kong dollars|hong kong dollar|hong kong $|hk$|hkd|hk dollars|hk dollar|hk $|hongkong$"
+            },
+            {
+                "Jamaican dollar",
+                "jamaican dollars|jamaican dollar|jamaican $|j$|jamaica dollars|jamaica dollar|jamaica $|jmd"
+            },
+            {"Kiribati dollar", "kiribati dollars|kiribati dollar|kiribati $"},
+            {"Liberian dollar", "liberian dollars|liberian dollar|liberian $|liberia dollars|liberia dollar|liberia $|lrd"},
+            {"Micronesian dollar", "micronesian dollars|micronesian dollar|micronesian $"},
+            {
+                "Namibian dollar",
+                "namibian dollars|namibian dollar|namibian $|nad|n$|namibia dollars|namibia dollar|namibia $"
+            },
+            {"Nauruan dollar", "nauruan dollars|nauruan dollar|nauruan $"},
+            {"Niue dollar", "niue dollars|niue dollar|niue $"},
+            {"Palauan dollar", "palauan dollars|palauan dollar|palauan $"},
+            {
+                "Pitcairn Islands dollar",
+                "pitcairn islands dollars|pitcairn islands dollar|pitcairn islands $|pitcairn island dollars|pitcairn island dollar|pitcairn island $"
+            },
+            {
+                "Solomon Islands dollar",
+                "solomon islands dollars|solomon islands dollar|solomon islands $|si$|sbd|solomon island dollars|solomon island dollar|solomon island $"
+            },
+            {"Surinamese dollar", "surinamese dollars|surinamese dollar|surinamese $|srd"},
+            {"New Taiwan dollar", "new taiwan dollars|new taiwan dollar|nt$|twd|ntd"},
+            {
+                "Trinidad and Tobago dollar",
+                "trinidad and tobago dollars|trinidad and tobago dollar|trinidad and tobago $|trinidad $|trinidad dollar|trinidad dollars|trinidadian dollar|trinidadian dollars|trinidadian $|ttd"
+            },
+            {"Tuvaluan dollar", "tuvaluan dollars|tuvaluan dollar|tuvaluan $"},
+            {"Dollar", "dollars|dollar|$"},
+            //Chinese yuan
+            {"Chinese yuan", "yuan|chinese yuan|renminbi|cny|rmb|￥"},
+            {"Fen", "fen"},
+            {"Jiao", "jiao"},
+            //Additional
+            {"Finnish markka", "suomen markka|finnish markka|finsk mark|fim|markkaa|markka"},
+            {"Penni", "penniä|penni"}
+        }.ToImmutableDictionary();
+
+        public static readonly ImmutableDictionary<string, string> CurrencyPrefixList = new Dictionary<string, string>
+        {
+            //Dollar Prefix
+            {"Dollar", "$"},
```

Comment:
I presume this works, but isn't $ part of the regex syntax and need to be escaped?